src/FormBuilder/resources/js/form-builder/src/styles/_block-editor.scss

```diff
@@ -116,6 +116,17 @@
 .block-editor-block-list__block {
     position: relative;
 
+    .wp-block {
+
+        &.is-selected {
+            padding: 1.5rem;
+            border-style: solid;
+            border-width: thin;
+            border-radius: 0.25rem;
+            border-color: var(--wp-admin-theme-color);
+        }
```

Comment:
My main hesitation with this is that it causes the blocks to become narrower when selected (due to the additional padding).  I think if we tap into the `::before` pseudo to add the ring without any movement this could work.  Here's something quick and dirty that I was thinking:


```suggestion
        &.is-selected::before {
            border-style: solid;
            border-width: thin;
            border-radius: 0.25rem;
            border-color: var(--wp-admin-theme-color);
            display: block;
            width: calc(100% + 1.5rem);
            content: '';
            position: absolute;
            height: calc(100% + 1.5rem);
            left: calc(-1.5rem / 2);
            top: calc(-1.5rem / 2);
        }
```

Reply:
I liked it, just change a bit to keep 1rem as "padding" on all sides.

![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/9ea301e4-3236-4716-9755-d46c195b5bcd)